bus/meson.build

```diff
@@ -43,6 +43,27 @@ else
                              required: true).path()
 endif
 
+launcher_args = [
+             '-DSYSCONFDIR="@0@"'.format(atspi_sysconfdir),
+             '-DDATADIR="@0@"'.format(atspi_datadir),
+             '-DDBUS_DAEMON="@0@"'.format(dbus_daemon),
+           ]
+
+if get_option('dbus_broker') != 'default'
+  launcher_args += '-DDBUS_BROKER="@0@"'.format(get_option('dbus_broker'))
+else
+  dbus_broker = find_program('dbus-broker-launch',
+                             '/sbin/dbus-broker-launch',
+                             '/usr/sbin/dbus-broker-launch',
+                             '/libexec/dbus-broker-launch',
+                             '/usr/libexec/dbus-broker-launch',
+                             '/usr/pkg/bin/dbus-broker-launch',
+                             required: false)
+  if dbus_broker.found()
+    launcher_args += '-DDBUS_BROKER="@0@"'.format(dbus_broker.path())
```

Comment:
This has no effect. `launcher_args` is not used anywhere.

Reply:
Ugh! Had an uncommitted hunk, not good! Sorry about that. Force-pushed with that hunk included.